website/src/theme/CodeBlock/index.js

```diff
@@ -84,7 +83,8 @@ export default ({
       const { Parser, packDataBytes } = await import('@taquito/michel-codec');
       const { RpcClient } = await import('@taquito/rpc');
       const { SaplingToolkit, InMemorySpendingKey, InMemoryViewingKey } = await import('@taquito/sapling');
-  
+      const TransportWebHID = (await import("@ledgerhq/hw-transport-webhid")).default;
```

Comment:
curious the reason behind this change have the package update what it export?

Reply:
I'm not sure, but as we update packages, some of them fail to be unable to be imported normally in the website. Should have something to do with ESM vs commonJS packaging

Reply:
If this is the only way it works, I understand that we have to
Not a blocker just trying to understand the root cause, unsure esm/commonjs is relevant in the context since @ledgerhq/hw-transport-webhid didn't upgrade